data/campaigns/Eastern_Invasion/scenarios/02_The_Escape_Tunnel.cfg

```diff
@@ -1,530 +1,625 @@
-        [/message]
-
-        # Lift shroud around the amulet
-        [scroll_to]
-            x,y=12,2
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=12,2
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        # Path leading through the troll cave
-        [scroll_to]
-            x,y=15,7
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=15,7
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=22,5
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=22,5
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=28,6
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=28,6
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=34,5
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=34,5
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [message]
-            speaker=Dacyn
-            message= _ "We must make haste to escape before the undead catch up."
-        [/message]
-
-        [message]
-            speaker=Gweddry
-            message= _ "Wait, before we go anywhere — who were those undead?"
-        [/message]
-
-        [message]
-            speaker=Dacyn
-            message= _ "I will explain later. For now, suffice to say they are much too powerful for us; our only hope is to head north."
-        [/message]
-    [/event]
-
-    [event]
-        name=moveto
-        [filter]
-            x,y=29,10
+#
+# Simple and easy, especially compared to S01.
+#
+# This is beatable without any combat or recruiting - just run straight through the dwarfish tunnels, and wade through the river.
+# More ambitious players can take the north path and raid the troll trezzur hole for some gold and a powerful item
+# REALLY ambitous players can fight their way through the troll leader for extra XP.
+#
+
+#define TURNS
+    {ON_DIFFICULTY 15 13 11}#enddef # undead arrival plus 3. I don't like tight turn limits, but couldn't think of another way to discourage farming the undead
+
+    [scenario]
+        id=02_The_Escape_Tunnel
+        name= _ "The Escape Tunnel"
+        map_file=02_The_Escape_Tunnel.map
+        turns={TURNS}
+        next_scenario=03_An_Unexpected_Appearance
+        victory_when_enemies_defeated=no
+
+        {UNDERGROUND}
+
+        [story]
+            [part]
+                story=_ "Screams and shouts echoed across the surface as Gweddry rushed his remaining men into the roughly hewn tunnel. A foul stench wafted up from below..."
```

Comment:
Sufficient just to say 'wafted from below'?

Reply:
@stevecotton Care to explain why you disagree?

Reply:
Just gut feeling, nothing more. Will comment instead of just downvoting in future.

Reply:
For me, it sounds redundant. From below already means the direction is 'up'. It's not a big deal - 'up' is often used redundantly in this way, especially in `en_US` (eg commit messages often say 'fix up' instead of just 'fix'), but I think PRs are the place to bring up these minor points.

Reply:
I think the redundancy is fine; "wafted up" feels like a single verb that can't/shouldn't be split

Reply:
That's because you're used to such redundancy in your cultural context. :p If I recall correctly you also like 'off of'.

Reply:
Well, yeah, but...

Reply:
> I think the redundancy is fine; "wafted up" feels like a single verb that can't/shouldn't be split

"waft up" is not a phrasal verb, no. Things can "waft" in any direction, for example "The scent wafted across the room."